src/engine/controls/cuecontrol.cpp

```diff
@@ -1801,30 +2090,138 @@ void CueControl::hotcueFocusColorNext(double v) {
     pCue->setColor(colorPalette.nextColor(*color));
 }
 
+void CueControl::setCurrentSavedLoopControl(HotcueControl* pControl) {
```

Comment:
I disagree, because the method is also used for resetting it (`nullptr`). The new name would be weird in that case. Also, the current saved loop control not reset when the loop is disabled, only when a *new* loop is set.

This means you can activate a saved loop, then use `reloop_toggle` to disable and then re-enable the loop and Mixxx automatically detects that re-activated the saved loop.